build/Dockerfile

```diff
@@ -1,24 +1,22 @@
 # This dockerfile is used to build Mattermost rtcd
 # A multi stage build, with golang:1.17.9-alpine used as a builder
-# and gcr.io/distroless/static as runner 
+# and gcr.io/distroless/static as runner
 FROM golang:1.17.9-alpine@sha256:5c2fcfeb58ad9d4948d94e7b2d0432a9bc38bee0f8dfb41d383f38a18b75c38d as builder
-RUN apk update && apk add make
 
-# Setup Golang directories structure and compile
-RUN mkdir -p /rtcd
-ADD . /rtcd
-WORKDIR /rtcd
-RUN make build
+# Setup directories structure and compile
+COPY . /src
+WORKDIR /src
+RUN apk add make=4.3-r0 --no-cache \
+    && make build
 
 # Shrink final image since we only need the rtcd binary
 # and use distroless container image as runner for security
 FROM gcr.io/distroless/static@sha256:d6fa9db9548b5772860fecddb11d84f9ebd7e0321c0cb3c02870402680cc315f as runner
-COPY --from=builder /rtcd/dist/rtcd /rtcd/rtcd
+COPY --from=builder /src/dist/rtcd-linux-amd64 /rtcd/rtcd
```

Comment:
if we follow strict the linux directory structure we should this in `/opt` directory.
_`/opt` is a directory for installing unbundled packages (i.e. packages not part of the Operating System distribution, but provided by an independent source), each one in its own subdirectory_

For example have directory `/opt/rtcd` which includes the binary in `/opt/rtcd/bin/<binary>`

Reply:
Agreed! Changing this to /opt!

Reply:
Rebased and pushed changes